stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -1076,7 +1087,7 @@ extension Unsafe${Mutable}RawBufferPointer {
   /// - Returns: The return value, if any, of the `body` closure parameter.
   @inlinable
   @_alwaysEmitIntoClient
-  public func withMemoryRebound<T, Result, E: Error>(
+  public func withMemoryRebound<T: ~Copyable, Result: ~Copyable, E: Error>(
```

Comment:
I think this is my mistake, but the list of generic parameters should be in the same order as they appear in the argument list. It is of no real consequence for an `@_aEIC` function, but when it's ABI we have to be careful. My point is that we probably gain from sticking to such a pattern.
Also some inlinable annotations remain on aEIC functions in this file.